x-pack/plugins/security/public/management/roles/__fixtures__/kibana_features.ts

```diff
@@ -253,4 +253,29 @@ export const kibanaFeatures = [
       },
     ],
   }),
+  createFeature({
+    id: 'with_require_all_spaces_for_feature_and_sub_features',
+    name: 'Require all spaces Features and Sub Features',
+    requireAllSpacesOnAllPrivilege: true,
```

Comment:
__note:__ I had to add a dedicated test feature to have `requireAllSpaces` for both main feature privilege and sub-feature privileges.